scrapers/base.py

```diff
@@ -1,45 +1,81 @@
+import os
 import boto3
 import logging
 import requests
-
+import json
 
 from bs4 import BeautifulSoup
 from time import gmtime, strftime
 
 
-from .config import AWS
+from scrapers.config import AWS
 
 log = logging.getLogger(__name__)
 
 
 class Scraper(object):
-    """
+    '''
     This is a base class inherited by other scraper classes.
-    """
+    '''
 
     def __init__(self):
         self.url = None
-        self.s3 = boto3.client("s3", **{
-            "aws_access_key_id": AWS["aws_access_key_id"],
-            "aws_secret_access_key": AWS["aws_secret_access_key"],
-            "region_name": AWS["region_name"]
+        self.s3 = boto3.client('s3', **{
+            'aws_access_key_id': AWS['aws_access_key_id'],
+            'aws_secret_access_key': AWS['aws_secret_access_key'],
+            'region_name': AWS['region_name']
         })
 
     def get_html_content(self, url):
-        """Returns the html content,\
-            a soup object.
+        '''Returns a soup object.
         Usage::
             pass the site url to the method.
         :param_train_data: the url of the site
         :rtype: the stories image,link, title.
-        """
+        '''
 
         try:
             res = requests.get(url)
             html = BeautifulSoup(res.content, 'html.parser')
             return html
 
         except Exception as err:
-            print "Url is not reachable. Error logged on: {}".format(
+            print 'Url is not reachable. Error logged on: {}'.format(
                 strftime("%Y-%m-%d %H:%M:%S", gmtime()))
             log.info(str(err))
+
+    def local_store(self, data, site_name):
+        '''Writes content from the website to file.
+        Usage::
+             file name to write data
+        :param_train_data: the filename
+        :rtype: outputs data to a file.
+        '''
+        with open(os.path.dirname(__file__) + '/data/'+site_name+'.json', 'r+') as output_file:
+            try:
+                # outputs data to the file.
+                existing_data = json.load(output_file)
+                output_file.truncate()
+            except:
+                existing_data = {'news': []}
+            finally:
+                if type(existing_data) is not dict:
+                    existing_data = {'news': []}
+                    # if data exists in the file, it adds data obtained by the next scraper.
+                existing_data['news'] += data
+        with open(os.path.dirname(__file__) + '/data/'+site_name+'.json', 'w') as output_file:
+            # writes data to the output file.
+            json.dump(existing_data, output_file, indent=2)
+
+    def aws_store(self, site_name, data):
```

Comment:
The parameters order looks different from `local_store`. Any reason why? Consistency is very important.